templates/index.html

```diff
@@ -237,7 +239,9 @@
 
         // {% if args.encryption_client_side || args.encryption_server_side || args.enable_readonly %}
         if (passwordField.value.trim() != "") {
+            // {% if !args.no_file_upload %}
```

Comment:
why is this commented?

Reply:
same question regarding the other commented bits

Reply:
it probably breaks JS parsing in code editors otherwise, and it's handled by the template system anyway

Reply:
ahh thanks (and thanks for the quick response)! I did not know that, I thought they where leftover commented bits from testing.